platform/linuxbsd/os_linuxbsd.cpp

```diff
@@ -34,6 +34,10 @@
 #include "main/main.h"
 #include "servers/display_server.h"
 
+#ifdef MODULE_REGEX_ENABLED
```

Comment:
You need to include `#include "modules/modules_enabled.gen.h"` before this, or it's not guaranteed to be defined.